htcondor/htcondor_query.py

```diff
@@ -293,6 +288,7 @@ def list2dict(list_data, attr_name):
                         # attribute exists in the fetched classad
                         dict_el[a] = list_el[a]
                 except Exception:
+                    logger.exception("Cannot find attribute for htcondor_query.list2dict")
```

Comment:
since this looks like a defensive `except`, can you add `# pragma: no cover` to the `except Exception:` line?